debezium-bom/pom.xml

```diff
@@ -30,7 +30,7 @@
         <version.mysql.binlog>0.25.1</version.mysql.binlog>
         <version.mongo.driver>4.2.1</version.mongo.driver>
         <version.sqlserver.driver>7.2.2.jre8</version.sqlserver.driver>
-        <version.oracle.driver>12.2.0.1</version.oracle.driver>
+        <version.oracle.driver>21.1.0.0</version.oracle.driver>
```

Comment:
Should `debezium-bom/pom.xml/README.md` have been updated as part of this change as well?

https://github.com/debezium/debezium/blob/7179fd6aa1273f014508c34a3952794eb1aca8ab/debezium-connector-oracle/README.md#L33

I had an issue while building this connector in IntelliJ IDEA where the IDE couldn't resolve the imports of the `oracle.streams.*` classes even though it did load the source jar. Eventually, I ended up specifying the version exactly as here, not as in README. Although, I'm not sure if it was the root cause.

Reply:
@morozov Thans for the report https://github.com/debezium/debezium/pull/2409

Reply:
Thx, good catch, @morozov!

That said, comments on closed PRs are easy to miss; so if you don't hear back in such case (e.g. I had missed this one, thx to @jpechane for replying), best is to reach out on the ML to raise awareness. Thx!